.github/risk_level_guide.md

```diff
@@ -0,0 +1,23 @@
+## Risk level guide
+
+**Low risk**: with high confidence the change won’t break the expected behaviors, or the change touches a non-critical area.
```

Comment:
I'm with @luke-pearson on this, the wording does seem a bit strange to me.

Reply:
How about 'confidence that the change won't unexpectedly change the behaviour'?

Reply:
Fixed in [cde76c0](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/cde76c03a3da2877ed66beb78c12876859a10939)

Reply:
In this context behaviour (singular) is better than behaviours (plural) (in both American and British spelling :-) ).

Reply:
Yes but this has it's own issues where the question becomes "the behaviour of what?". We seem to be tiptoeing around a somewhat clearer definition of which is to say "With high confidence that the change won't introduce bugs". I think in this case we can just leverage the definition of "bug" instead of being a bit ambiguous with behaviour.